Mathlib/RingTheory/IntegralClosure/IsIntegral/Basic.lean

```diff
@@ -179,6 +179,7 @@ theorem isIntegral_iff_isIntegral_closure_finite {r : B} :
   rcases hr with ⟨s, _, hsr⟩
   exact hsr.of_subring _
 
+@[stacks 09GH]
 theorem fg_adjoin_of_finite {s : Set A} (hfs : s.Finite) (his : ∀ x ∈ s, IsIntegral R x) :
```

Comment:
I think either 02JJ or 00GM is more suitable here.

Reply:
Thanks! Fixed!